docs/api.md

```diff
@@ -1067,27 +1153,249 @@ Example Response Body:
+| position        | The balance in position token (e.g. PBTC).                                     |
+| cachedMargin    | This is the last stored margin value                                           |
+| pendingMargin   | This is the (pending) component of the margin when a fill is still pending     |
+| pendingPosition | This is the (pending) component of the position when a fill is still pending   |
+| indexValue      | The value of the global index from the last interaction with the account       |
+| indexTimestamp  | The timestamp when the index value was set                                     |
+
+
+### GET `/v1/perpetual-accounts`
+
+Description:
+This endpoint returns balances for all perpetual accounts.
+
+Query Params:
+| Field Name     | Description                                                     |
+|----------------|-----------------------------------------------------------------|
+| isLiquidatable | If set to true, returns accounts that are yet to be liquidated  |
+
+Example Response Body:
+
+```json
+{
+  "accounts": [
+    {
+      "owner": "0x000f7f22bfc28d940d4b68e13213ab17cf107790",
+      "market": "PBTC-USDC",
+      "orderNumber": "1011196300330012",
+      "cachedMargin": "36759577102",
+      "margin": "36759577102",
+      "position": "1201838526",
+      "cachedIndexValue": "1.211642619304704688",
+      "indexTimestamp": "1590100515"
+    },
+    {
+      "owner": "0x003f480be5b68c5e863d2990d043f3b58f64473f",
+      "market": "PBTC-USDC",
+      "orderNumber": "1010984800560005",
+      "cachedMargin": "368456014",
+      "margin": "368468282",
+      "position": "-3000000",
+      "cachedIndexValue": "1.207553221962107213",
+      "indexTimestamp": "1590072186"
+    }
+  ]
+}
+```
+
+#### Account Response Object
+
+| Field Name      | Description                                                                    |
+|-----------------|--------------------------------------------------------------------------------|
+| owner           | The user's wallet address.                                                     |
+| balances        | An object with the user's balances for each market.                            |
+| margin          | This is calculated as `cachedMargin- (indexValue - globalIndexValue)*position` |
+| position        | The balance in position token (e.g. PBTC).                                     |
+| cachedMargin    | This is the last stored margin value                                           |
+| pendingMargin   | This is the (pending) component of the margin when a fill is still pending     |
+| pendingPosition | This is the (pending) component of the position when a fill is still pending   |
+| indexValue      | The value of the global index from the last interaction with the account       |
+| indexTimestamp  | The timestamp when the index value was set                                     |
+
+
+# Solo and Perpetual endpoints
```

Comment:
Is this meant to be the endpoint documentation? I think you may be missing the line:

```
### GET `/v1/standard-actions`
```

About the heading level -- currently this is an H1, and we currently have the following H2 headings:

```
## Introduction
## Trading Endpoints (Solo and Perpetual)
## Solo Endpoints
## Perpetual Endpoints
## Deprecated APIs
```

When I reorganized the headings earlier, I kept the name “Trading Endpoints” but it's not very meaningful. I think it would be fine to rename that heading to “Solo and Perpetual Endpoints” and then move this section so it can go together with the other Solo+Perpetual endpoints.